commit 9e4a591e72417817a234e6e82e11963745e10095
Author: Ron Lieberman <ron.lieberman@amd.com>
Date:   Thu Oct 8 21:13:58 2020 -0500

    Fix openmp error message output for no_rocm_device_lib
    
    we were asserting due to missing arguments for missing rocm library message
    when trying to use -offload-arch=gfx906
    
    Change-Id: I0fce362ff2a091cbfecfc65bf46ce9b454807b9d

diff --git a/clang/lib/Driver/ToolChains/AMDGPUOpenMP.cpp b/clang/lib/Driver/ToolChains/AMDGPUOpenMP.cpp
index 809522059aca..834c9a6cb1ef 100644
--- a/clang/lib/Driver/ToolChains/AMDGPUOpenMP.cpp
+++ b/clang/lib/Driver/ToolChains/AMDGPUOpenMP.cpp
@@ -457,7 +457,7 @@ void AMDGPUOpenMPToolChain::addClangTargetOptions(
 
   } else {
     if (!RocmInstallation.hasDeviceLibrary()) {
-      getDriver().Diag(diag::err_drv_no_rocm_device_lib);
+      getDriver().Diag(diag::err_drv_no_rocm_device_lib) << 0;
       return;
     }
 
@@ -485,7 +485,7 @@ void AMDGPUOpenMPToolChain::addClangTargetOptions(
 
     std::string LibDeviceFile = RocmInstallation.getLibDeviceFile(CanonArch);
     if (LibDeviceFile.empty()) {
-      getDriver().Diag(diag::err_drv_no_rocm_device_lib) << GpuArch;
+      getDriver().Diag(diag::err_drv_no_rocm_device_lib) << 1 << GpuArch;
       return;
     }
 
